icons/monzo.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>monzo  icon</title><path d="M24 6.49L20 2.5l-.15-.12-.28.01-.15.11-7.41 7.41v8.54l5.96-5.95v11.12l.03.14.06.11.13.08.11.05h.15l.14-.03.09-.08 5.07-5.07.13-.19.07-.17.05-.24V6.49zM6.03 12.5l5.96 5.95V9.91L4.58 2.5l-.15-.11-.28-.01L4 2.5 0 6.49v11.73l.05.24.07.17.13.19 5.07 5.07.09.08.14.03h.15l.11-.05.13-.08.06-.11.03-.14V12.5z"/></svg>
```

Comment:
Title should use capital letters as well and it has two spaces rather then one 😉 

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Monzo icon</title><path d="M24 6.49L20 2.5l-.15-.12-.28.01-.15.11-7.41 7.41v8.54l5.96-5.95v11.12l.03.14.06.11.13.08.11.05h.15l.14-.03.09-.08 5.07-5.07.13-.19.07-.17.05-.24V6.49zM6.03 12.5l5.96 5.95V9.91L4.58 2.5l-.15-.11-.28-.01L4 2.5 0 6.49v11.73l.05.24.07.17.13.19 5.07 5.07.09.08.14.03h.15l.11-.05.13-.08.06-.11.03-.14V12.5z"/></svg>
```